static/sass/_pattern_card.scss

```diff
@@ -43,4 +43,66 @@
       text-decoration: underline;
     }
   }
+
+  %resources-card-header {
+    border-radius: 2px;
+    padding: $sp-small $sp-medium;
+  }
+
+  .p-card--resources {
+    @extend .p-card--highlighted;
+    display: flex;
+    flex-direction: column;
+    padding: 0;
+
+    > .p-card__content {
+      border-top: 1px dotted $color-mid-light;
+      margin: 0 $sp-x-small;
+      padding: $sp-x-small;
+
+      > h3, p {
+        margin: $sp-x-small 0;
+      }
+
+      > a {
+        display: block;
+        margin: $sp-x-small 0 $sp-medium 0;
+        min-height: 11rem;
+
+        @media only screen and (min-width: $breakpoint-medium) {
+          margin: $sp-x-small 0;
+        }
+      }
+    }
+
+    > .p-card__footer {
+      border-top: 1px dotted $color-mid-light;
+      font-size: .875rem;
+      margin: auto $sp-x-small 0;
+      max-width: inherit;
+      padding: $sp-medium $sp-x-small;
+    }
+  }
+
+  .p-card__header {
+    &--cloud-and-server {
+      @extend %resources-card-header;
+      border-top: 3px solid #2b939c;
+    }
+
+    &--desktop {
+      @extend %resources-card-header;
+      border-top: 3px solid #66a2c1;
```

Comment:
Desktop and Cloud and server colours are very similar.

Reply:
The secondary colour palette has been signed off already. I believe Spencer and Frankie will look into it later this/next week, but for now I think we can leave it. I do agree though that with just the top border colour they look very similar.